cmd/client-admin.go

```diff
@@ -79,11 +79,11 @@ func newAdminFactory() func(config *Config) (*madmin.AdminClient, *probe.Error)
 			var transport http.RoundTripper = &http.Transport{
 				Proxy: http.ProxyFromEnvironment,
 				DialContext: (&net.Dialer{
-					Timeout:   30 * time.Second,
-					KeepAlive: 30 * time.Second,
+					Timeout:   60 * time.Second,
+					KeepAlive: 60 * time.Second,
 				}).DialContext,
 				MaxIdleConns:          100,
-				IdleConnTimeout:       90 * time.Second,
+				IdleConnTimeout:       300 * time.Second,
```

Comment:
increasing this is not needed @wlan0 is there a good reason to increase this value? I don't think this should adversely affect your implementation because the server will always send data anyways and it won't trigger this.

Reply:
This isn't needed anymore. It was a debugging session where I introduced this change, and missed removing it later.